doc/releases/changelog-dev.md

```diff
@@ -156,6 +159,10 @@
 
 <h3>Breaking changes 💔</h3>
 
+* `QNode` and `qml.execute` now forbid certain keyword arguments from being passed positionally.
+  This helps prevent keyword arguments from being passed in the wrong order.
+  [(#6610)](https://github.com/PennyLaneAI/pennylane/pull/6610)
+
```

Comment:
Is it intentional to have 2 changelog entries?

Reply:
improvement versus breaking change?

Reply:
But actually, this isn't really a breaking change of importance. I'll remove it from here.